spec/rubocop/cop/style/access_modifier_declarations_spec.rb

```diff
@@ -697,10 +697,17 @@ class TestOne
           end
 
           class TestTwo
+            #{access_modifier} def foo; end
           end
+        RUBY
+      end
 
-          class TestThree
-          end
+      it "does not register an offense for #{access_modifier} without method definitions" do
+        expect_no_offenses(<<~RUBY)
+          #{access_modifier}
+          1
+          foo
+          bar(1, 2)
```

Comment:
Isn't the following a simpler reproduction that focuses on the main issue?

```suggestion
          #{access_modifier}
```